pinot-common/src/main/java/com/linkedin/pinot/common/metadata/segment/IndexLoadingConfigMetadata.java

```diff
@@ -32,10 +32,14 @@
   public static final String KEY_OF_LOADING_INVERTED_INDEX = "metadata.loading.inverted.index.columns";
   public static final String KEY_OF_SEGMENT_FORMAT_VERSION = "segment.format.version";
   public static final String KEY_OF_ENABLE_DEFAULT_COLUMNS = "enable.default.columns";
+  public static final String KEY_OF_STAR_TREE_FORMAT_VERSION = "startree.format.version";
+
   private final Set<String> _loadingInvertedIndexColumnSet = new HashSet<String>();
-  private final String DEFAULT_SEGMENT_FORMAT = "v1";
+  private final String DEFAULT_SEGMENT_FORMAT = "V1";
+  private static final String DEFAULT_STAR_TREE_FORMAT = "V1";
```

Comment:
Are you sure it's capital 'v' (both for DEFAULT_SEGMENT_FORMAT and DEFAULT_STAR_TREE_FORMAT)? It's not the same as in CommonConstants